### DISCUSSION TITLE
plot a miller(-1,1,2) in a {1 0 0} PF

### DISCUSSION THREAD

**Discussion by SingPr** [mtex-toolbox/mtex#1580] [Ask Anything]
Hi everyone,
   I am a beginner of Mtex, I want to plot a Miller(-1,1,2) in a {1 0 0} PF, so, how to write the code? I 
```matlab
m = Miller(-1,1,2,cs,'hkl')
plot (m,'upper','MarkerColor','green','Marksize',1.5)
```
 Is it right? I project {1 0 0} onto (-1 1 2,) and draw a {1 0 0} PF, but it doesn't work. 
Thanks 
Sing

**Comment by ralfHielscher** [comment]
Hi Sing,
what you are requesting does not really makes sense.
A {1 0 0} pole figure is a plot aligned with you specimen, i.e., you can annotate to it specific directions of your same, e.g. TD, RD, ...
Furthermore, you can visualize in it orientations, namely exactly at those positions where the {1 0 0} direction is located with respect to your specimen coordinates. This means you always visualize an crystal direction (Miller) subject to a certain orientation.

Maybe, what you really want to to is to overly two pole figures?

Ralf.

**Reply by SingPr** [reply]
Thanks, Ralf, I know that.
sing

**Reply by SingPr** [reply]
Hi Ralf.
    Now, if I plot a Miller(-1,1,2) in a {1 1 0} PF or other PFs, just overly two PFs, so, is it still as you said above?
Thanks
Sing

**Reply by ralfHielscher** [reply]
Sorry Sing,
you need to better understand what a pole figures is. Your question really makes no sense. Maybe this https://mtex-toolbox.github.io/OrientationPoleFigure.html helps.

Ralf.
